lib/types/SVGComponent.ts

```diff
@@ -0,0 +1 @@
+export type SVGComponent = React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
```

Comment:
@KATT are you ok with this? I believe we can reuse this type more often vs. only in Button